charts/orchestrator/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.0-rc1
+version: 1.2.0-rc2
```

Comment:
should this be bumped to rc3?

Reply:
Why? because the operator is already 1.2.0-rc2? I thought the chart and operator versions were not meant to be matched.

Reply:
My wrong, I thought version `1.2.0-rc2` was already released.
I see it wasn't - https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/docs/index.yaml#L15
So this PR is good as is.